scout/server/config.py

```diff
@@ -127,3 +127,8 @@
 RANK_MODEL_LINK_POSTFIX = "-.ini"
 SV_RANK_MODEL_LINK_PREFIX = "https://raw.githubusercontent.com/Clinical-Genomics/reference-files/master/rare-disease/rank_model/svrank_model_-v"
 SV_RANK_MODEL_LINK_POSTFIX = "-.ini"
+
+# BioNano Access Server extension
```

Comment:
Add a link to somewhere where who's interested to add the extension can go and check out how?